app/controllers/concerns/sufia/files_controller_behavior.rb

```diff
@@ -59,6 +61,9 @@ def load_resource_from_solr
 
     # routed to /files/new
     def new
+      find_collections_with_edit_access
+      default_option = SolrDocument.new(id: -1, title_tesim: t("sufia.upload.collection.select_default_option"))
```

Comment:
I might like to see this code (lines 65-66) live in SelectsCollections in hydra-collections.